lib/eth/key/decrypter.rb

```diff
@@ -0,0 +1,135 @@
+# Copyright (c) 2016-2022 The Ruby-Eth Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "json"
+require "scrypt"
+
+class Eth::Key::Decrypter
+  include Eth::Util
```

Comment:
I would prefer not to include other modules unless they are submodules of the current. It improves readability if you always type `Util.bin_to_hex` knowing you call a function of a different module rather than something within the same class.